PDFTron/webviewer-ui
# Feature/measurement overlay updates

Requires: This[ branch ](https://github.com/XodoDocs/webviewer/pull/571)from webviewer if not already merged. 

QA Steps: 
- Run webviewer and start using the line measurement annotation tool. 
- In the measurement overlay in the bottom right corner you will now notice the length property is an input. 
- Try changing the value and see how the line redraws. If you enter a large value that causes the line to go off screen, it should automatically resize to stay within bounds when you click anywhere on the document or tab away from the input box.
